diff --git a/src/OplPcTools/GameConfiguration.cpp b/src/OplPcTools/GameConfiguration.cpp
index 7f0151c..22bc5c3 100644
--- a/src/OplPcTools/GameConfiguration.cpp
+++ b/src/OplPcTools/GameConfiguration.cpp
@@ -224,7 +224,7 @@ void GameConfiguration::save(const GameConfiguration & _config, const QString &
     QFile src_file(_filename);
     QFile tmp_file(_filename + ".tmp");
     QFileInfo(tmp_file).absoluteDir().mkpath(".");
-    src_file.open(QFile::ReadOnly | QFile::Text);
+    (void)src_file.open(QFile::ReadOnly | QFile::Text);
     openFile(tmp_file, QFile::WriteOnly | QFile::Truncate | QFile::Text);
     QSet<QString> written_keys;
     for(; src_file.isOpen(); )
