From 02be2b8775cc5bcf1924c9fb2cbf5caa5a906e7c Mon Sep 17 00:00:00 2001
From: rtkt <rtkt@protonmail.com>
Date: Sat, 4 Feb 2023 14:55:03 +0700
Subject: [PATCH] Renamed QT5_DIR to Qt5_DIR

---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8625548..3b88cf3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,14 +49,14 @@ endif()
 # Libraries
 #######################
 
-set(QT5_DIR "" CACHE PATH "Provides a path to search Qt5 installation")
-find_package(Qt5Core HINTS ${QT5_DIR})
+set(Qt5_DIR "" CACHE PATH "Provides a path to search Qt5 installation")
+find_package(Qt5Core HINTS ${Qt5_DIR})
 if(Qt5Core_FOUND)
     message(STATUS "Qt version: ${Qt5Core_VERSION_STRING}")
 else()
-    message(FATAL_ERROR "ERROR: Qt not found. Please specify a path to Qt installation using the QT5_DIR variable.")
+    message(FATAL_ERROR "ERROR: Qt not found. Please specify a path to Qt installation using the Qt5_DIR variable.")
 endif()
-find_package(Qt5 COMPONENTS Widgets Network LinguistTools REQUIRED HINTS ${QT5_DIR})
+find_package(Qt5 COMPONENTS Widgets Network LinguistTools REQUIRED HINTS ${Qt5_DIR})
 
 #######################
 # Sources
-- 
2.39.1

